trillian/integration/ct_hammer/main.go

```diff
@@ -158,6 +160,10 @@ func main() {
 		if err != nil {
 			glog.Exitf("Failed to create client pool: %v", err)
 		}
+		var limiter integration.Limiter
```

Comment:
You could pop this into a `limiterFromFlags`  method which returns a thing or nil?

Reply:
Done.